src/index.ts

```diff
@@ -46,22 +46,29 @@ const observeGitInfo = async (
 
 async function serverChannel(
   channel: Channel,
-  { projectToken: initialProjectToken }: { projectToken: string }
+  {
+    projectToken: initialProjectToken,
+    buildScriptName,
+  }: { projectToken?: string; buildScriptName?: string }
 ) {
+  console.log({ initialProjectToken, buildScriptName });
```

Comment:
@tmeasday, before we forget, could you remove this line so we don't accidentally expose the information in the CLI? I'm assuming that this was added for testing purposes, correct?

Reply:
```suggestion
```

Reply:
Good pickup, thanks @jonniebigodes